log/Logger.js

```diff
@@ -1,26 +1,48 @@
-const moment = require('moment')
-const winston = require('winston')
+const util = require('util');
+const {createLogger, format, transports} = require('winston');
+const {errors, json, metadata, timestamp} = format;
 
-const container = new winston.Container()
-
-function timestamp() {
-  return moment().format('YYYY-MM-DDTHH:mm:ssZ')
-}
-
-function transport(label) {
-  return new winston.transports.Console({
-    level: (process.env.LOG_LEVEL || 'INFO').toLowerCase(),
-    json: process.env.JSON_PRINT || false,
-    timestamp: timestamp,
-    label: label
-  })
-}
+const customMessageFormat = format((info) => {
+  const {message, metadata} = info;
+  if (metadata && typeof metadata === 'object' && Object.keys(metadata).length) {
+    info.message = message + util.inspect(metadata);
```

Comment:
The message will contain any dynamic data that is added to the logger statement for e.g.
```
logger.info('logging an object', object);
```
However, if we want to log non-objects, we will have to do something like this
```
logger.info(`Test stattus: ${status}`)
```

hope this is ok?

Reply:
done